pipeline_api/src/main/java/com/splicemachine/pipeline/client/MergingWriteStats.java

```diff
@@ -22,19 +22,19 @@
  *         Date: 2/5/14
  */
 public class MergingWriteStats implements WriteStats {
-    private final Counter writtenCounter;
-    private final Counter retryCounter;
-    private final Counter thrownErrorsRows;
-    private final Counter retriedRows;
-    private final Counter partialRows;
-    private final Counter partialThrownErrorRows;
-    private final Counter partialRetriedRows;
-    private final Counter partialIgnoredRows;
-    private final Counter partialWrite;
-    private final Counter ignoredRows;
-    private final Counter catchThrownRows;
-    private final Counter catchRetriedRows;
-    private final Counter regionTooBusy;
+    public final Counter writtenCounter;
```

Comment:
this way it's accessible from BulkWriteAction. Otherwise we have to re-create the same variables over at BulkWriteReader again. Actually i think we only need MergingWriteStats, not WriteStats and SimpleWriteStats. It's a bit overengineered for something that is basically just a collection of counters in my opinion.